conda-store-server/tests/test_server.py

```diff
@@ -335,6 +336,60 @@ def test_create_specification_unauth(testclient):
     assert r.status == schema.APIStatus.ERROR
 
 
+# Only testing size values that will always cause errors. Smaller values could
+# cause errors as well, but would be flaky since the test conda-store state
+# directory might have different lengths on different systems, for instance,
+# due to different username lengths.
+@pytest.mark.parametrize(
+    "size",
+    [
+        # OSError: [Errno 36] File name too long
+        255,
+        # OSError: [Errno 36] File name too long
+        256,
```

Comment:
These error messages the code used to throw before I moved the `build_path` to the start of the function in `build_conda_environment`